server/src/test/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -582,6 +582,13 @@ public final void createIndex(String... names) {
         }
     }
 
+    /**
+     * creates an index with the given setting
+     */
+    public final void createIndex(String name, Settings indexSettings) {
+        assertAcked(prepareCreate(name).setSettings(indexSettings));
+    }
+
```

Comment:
I think it would be nicer if we would instead stick to using SQL instead of relying more on ES api.

Ideally we'd get rid of the `prepareCreate` eventually.